feature/experimental/tunnel/otg_tests/tunnel_interface_based_disable_enable_test/tunnel_interface_disable_enable_test.go

```diff
@@ -0,0 +1,1558 @@
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r1Fti7Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+	}
+
+	// Configure the interfaces.
+	for _, intf := range dutIntfs {
+		t.Logf("Configure DUT interface %s with attributes %v", intf.intfName, intf)
+		i := &oc.Interface{
+			Name:        ygot.String(intf.intfName),
+			Description: ygot.String(intf.desc),
+			Type:        oc.IETFInterfaces_InterfaceType_ethernetCsmacd,
+			Enabled:     ygot.Bool(true),
+		}
+
+		// configure ipv4 address
+		i.GetOrCreateEthernet()
+		i4 := i.GetOrCreateSubinterface(0).GetOrCreateIpv4()
+		a := i4.GetOrCreateAddress(intf.ipAddr)
+		a.PrefixLength = ygot.Uint8(intf.ipv4mask)
+
+		// configure ipv6 address
+		i6 := i.GetOrCreateSubinterface(0).GetOrCreateIpv6()
+		b := i6.GetOrCreateAddress(intf.ipv6Addr)
+		b.PrefixLength = ygot.Uint8(intf.ipv6mask)
+		gnmi.Replace(t, dut, gnmi.OC().Interface(intf.intfName).Config(), i)
+	}
+}
+
+func ConfigureTunnelInterface(t *testing.T, intf string, tunnelSrc string, tunnelDst string, dut *ondatra.DUTDevice) {
+
+	// IPv4 tunnel source and destination configuration
+	t.Logf("Push the IPv4/IPv6 tunnel endpoint config:\n%s", dut.Vendor())
+	var config string
+	switch dut.Vendor() {
+	case ondatra.JUNIPER:
+		config = ConfigureTunnelEndPoints(intf, tunnelSrc, tunnelDst)
+		t.Logf("Push the CLI config:\n%s", config)
+
+	default:
+		t.Errorf("Invalid Tunnel endpoint configuration")
+	}
+	gnmiClient := dut.RawAPIs().GNMI().Default(t)
+	gpbSetRequest, err := buildCliConfigRequest(config)
+	if err != nil {
+		t.Fatalf("Cannot build a gNMI SetRequest: %v", err)
+	}
+	t.Log("gnmiClient Set CLI config")
+	if _, err = gnmiClient.Set(context.Background(), gpbSetRequest); err != nil {
+		t.Fatalf("gnmiClient.Set() with unexpected error: %v", err)
+	}
+}
+
+func sortPorts(ports []*ondatra.Port) []*ondatra.Port {
+	sort.Slice(ports, func(i, j int) bool {
+		idi, idj := ports[i].ID(), ports[j].ID()
+		li, lj := len(idi), len(idj)
+		if li == lj {
+			return idi < idj
+		}
+		return li < lj // "port2" < "port10"
+	})
+	return ports
+}
+
+func ConfigureTunnelEndPoints(intf string, tunnelSrc string, tunnelDest string) string {
+
+	return fmt.Sprintf(`
+	interfaces {
+	%s {
+		unit 0 {
+			tunnel {
+				encapsulation gre {
+					source {
+						address %s;
+					}
+					destination {
+						address %s;
+					}
+				}
+			}
+		}
+	}
+	}`, intf, tunnelSrc, tunnelDest)
+
+}
+
+func ConfigureAdditionalIPv4AddressonLoopback(address string) string {
+
+	return fmt.Sprintf(`
+	interfaces {
+
+    lo0 {
+        unit 0 {
+            family inet {
+                address %s;
+            }
+        }
+    }
+}`, address)
+
+}
+
+func ConfigureAdditionalIPv6AddressonLoopback(address string) string {
+
+	return fmt.Sprintf(`
+	interfaces {
+
+    lo0 {
+        unit 0 {
+            family inet6 {
+                address %s;
+            }
+        }
+    }
+}`, address)
+
+}
+
+func ConfigureTunnelTerminationOption(interf string) string {
+
```

Comment:
delete blank line, and so on for all other "first lines of methods" that are blank